lib/jekyll.rb

```diff
@@ -115,9 +116,11 @@ def configuration(override = Hash.new)
     # timezone - the IANA Time Zone
     #
     # Returns nothing
+    # rubocop:disable Style/AccessorMethodName
     def set_timezone(timezone)
-      ENV['TZ'] = timezone
+      ENV["TZ"] = timezone
     end
+    # rubocop:enable Style/AccessorMethodName
```

Comment:
@parkr @envygeeks since it continues for the entire file, let it be there ? so new method added would be checked by rubocop